.github/workflows/conformance-externalworkloads.yaml

```diff
@@ -113,7 +113,8 @@ jobs:
           # Use complete matrix in case of scheduled run
           # main -> event_name = schedule
           # other stable branches -> PR-number starting with v (e.g. v1.14)
-          if [[ "${{ github.event_name }}" == "schedule" || "${{ inputs.PR-number }}" == v* ]];then
+          VERSIONS=$(echo "${{ inputs.extra-args }}" | awk -F'=' '{print $2}')
```

Comment:
I guess this breaks if you specify both the version and the channel args at the same time (same for the conformance-gke equivalent).